docs/user-guide/release-notes/2.8.x.rst

```diff
@@ -2,6 +2,20 @@
 Pulp Puppet 2.8 Release Notes
 =============================
 
+Pulp Puppet 2.8.4
+=================
+
+- A migration included in this release fixes an issue that was caused by incorrect Puppet module's
```

Comment:
Linking to the issue would be good. It could replace `an issue` with `issue xxxx`.